lib/residence/residence_screen.dart

```diff
@@ -0,0 +1,564 @@
+                  child: Icon(Icons.search,size: 25,),
+                ),
+                Text('物件',style: TextStyle(fontSize: 10),),
+              ],
+            ),
+          ), //テキスト//アイコン
+        ),
+      ),
+    );
+  }
+
+  Widget _buildBody(BuildContext context) {
+    return SingleChildScrollView(
+      child: Container(
+        color: Colors.yellow[50],
+        child: Column(
+          children: [
+            Padding(
+              padding: const EdgeInsets.only(top: 40),
+              child: Material(
+                elevation: 6.0,
+                child: Row(
+                  children: [
+                    const Padding(
+                      padding: EdgeInsets.only(left: 8, bottom: 3, right: 7),
+                      child: Chip(
+                        label: Text('カウルのおすすめ',
+                            style: TextStyle(color: Colors.teal,fontWeight: FontWeight.w700)),
+                      ),
+                    ),
+                    Badge(
+                      badgeContent: const Text('1',
+                          style: TextStyle(color: Colors.white)),
+                      position: BadgePosition.topEnd(top: -4, end: -4),
+                      child: const Chip(
+                        label: Text(
+                            style: TextStyle(fontWeight: FontWeight.w700), 'リフォーム済み'),
+                      ),
+                    ),
+                    const Spacer(), //間をあけれる
+                    const Padding(
+                      padding: EdgeInsets.only(right: 8, bottom: 3),
+                      child: Icon(
+                        Icons.add_circle,
+                        size: 45,
+                        color: Colors.teal,
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+            ),
+            Padding(
+              padding: const EdgeInsets.only(top: 10),
+              child: Column(
+                children: [
+                  Card(
+                    elevation: 3,
+                    child: Column(
+                      children: [
+                        Padding(
+                          padding: const EdgeInsets.only(top: 12),
+                          child: Row(
+                            children: const [
+                              Padding(
+                                padding: EdgeInsets.only(left: 20.0),
+                                child: Text('カウルのおすすめ    ',
+                                    style:
+                                        TextStyle(fontWeight: FontWeight.w900)),
+                              ),
+                              Text('新着3件',
+                                  style: TextStyle(
+                                      color: Colors.deepOrange,
+                                      fontSize: 12,
+                                      fontWeight: FontWeight.bold)),
+                              Spacer(),
```

Comment:
ここだけじゃなく他の箇所でもそうなのですが、コンマ(,)をつけてフォーマッターかけてほしいです！以下の用な感じになるようにですね！

```suggestion
                                      fontSize: 12,
                                      fontWeight: FontWeight.bold,
                                        ),
                                      ),
                              Spacer(),
```

Reply:
ありがとうございます！全て修正しました！